src/MobileUI/Features/Quiz/QuizPage.xaml

```diff
@@ -20,17 +21,27 @@
                           RowDefinitions="Auto" 
                           x:Name="CarouselSection">
                         <Grid.Triggers>
-                            <DataTrigger TargetType="Grid" Binding="{Binding CarouselQuizzes.Count}" Value="0">
+                            <MultiTrigger TargetType="Grid">
+                                <MultiTrigger.Conditions>
+                                    <BindingCondition Binding="{Binding CarouselQuizzes.Count}" Value="0" />
+                                    <BindingCondition Binding="{Binding IsBusy}" Value="False" />
```

Comment:
Binding to `IsBusy` we loose the animation when carousel and list slide from top and bottom of the screen. Do we need this binding?